src/models/record.go

```diff
@@ -15,10 +15,14 @@ type Record struct {
 	Json      map[string]interface{}
 }
 
-func (r *Record) FormatTimestamp() string {
+func (r *Record) FormatTimestampDay() string {
 	return r.Timestamp.Format("2006-01-02")
 }
 
+func (r *Record) FormatTimestampHour() string {
+	return r.Timestamp.Format("2006-01-02-15")
```

Comment:
I'm not sure this is the best format; it is somewhat contrary to ISO8601, which suggests that date and time should be either delimited by a `T` or omit the character at all. You could change the third `-` by a `T`:

```suggestion
	return r.Timestamp.Format("2006-01-02T15")
```